docs/source/release_notes/release-notes.rst

```diff
@@ -73,6 +73,9 @@ develop
           Previously, the table reference was omitted, such that one might receive lines of the form ``Call to KVS.getRange on table RangeRequest{reverse=false} with range 1504 took {} ms.``.
           (`Pull Request <https://github.com/palantir/atlasdb/pull/2474>`__)
 
+    *   - |improved|
+        - Safe and Unsafe args were used for Cassandra* classes.
```

Comment:
Perhaps: "Specified which logs from Cassandra* classes were Safe or Unsafe for collection, improving the data that we can collect for debugging purposes"